core/src/main/java/com/linecorp/armeria/common/AbstractHttpRequestBuilder.java

```diff
@@ -411,42 +425,142 @@ private RequestHeaders requestHeaders() {
 
     private String buildPath() {
         checkState(path != null, "path must be set.");
-        final StringBuilder pathBuilder = new StringBuilder(path);
+
+        boolean hasQueryString = false;
         if (!disablePathParams) {
```

Comment:
Don't we need to check that a path contains a query string regardless of path params?
The following test is failed:
```
final HttpRequest req = HttpRequest.builder()
                                   .get("/query?foo=bar")
                                   .disablePathParams()
                                   .queryParam("alice", "bob")
                                   .build();
// Expected :"/query?foo=bar&alice=bob"
// Actual   :"/query?foo=bar?alice=bob"
assertThat(req.path()).isEqualTo("/query?foo=bar&alice=bob");
```

Reply:
Fixed :scream: